### DISCUSSION TITLE
Package updates not persistent?

### DISCUSSION THREAD

**Discussion by Centurion-Rome** [vladmandic/sdnext#1018] [Q&A] [answered]
Everytime I start I got message like:

> 10:26:48-354725 WARNING  Package wrong version: tqdm 4.64.1 required 4.65.0
> 10:26:48-355725 INFO     Installing package: tqdm==4.65.0
> 10:26:54-788457 WARNING  Package wrong version: diffusers 0.14.0 required 0.16.1
> 10:26:54-789471 INFO     Installing package: diffusers==0.16.1

My command line params on windows: --upgrade
makeing  "git pull" and install requirements didn't help.

Should it be updated packages present by next start?
How can I update them so programm will not upgrade them by every start?

**Comment by vladmandic** [comment, marked as answer]
That means that some extension you've installed is making those changes without asking.
When you install extension, you give it full rights, I cannot monitor what extension does.

**Reply by Centurion-Rome** [reply]
You are right, it seems to be extension releated => so ok for me ;)
thank you for your fast answer!
